moveit_core/online_signal_smoothing/src/butterworth_filter.cpp

```diff
@@ -100,12 +100,12 @@ bool ButterworthFilterPlugin::initialize(rclcpp::Node::SharedPtr node, moveit::c
 {
   node_ = node;
   num_joints_ = num_joints;
+  auto param_listener_ = std::make_unique<online_signal_smoothing::ParamListener>(node_);
```

Comment:
The parameter is only read and set once when `initialize()` is called.
Is this the expected behaviour? 
or should this also check for updates in the parameter and reset the coefficient ?